SportHub/src/modules/Header/components/Header.js

```diff
@@ -0,0 +1,29 @@
+import AdminHeader from "../../AdminHeader";
+import UserHeader from "../../UserHeader/components/UserHeader";
+import NotLoginedHeader from "../../NotLoginedHeader/componets/NotLoginedHeader";
+import {useAuthStore} from "../../../store/useAuthStore";
+
+export default function Header()
+{
+    const { userData } = useAuthStore()
+    const fn = () =>
+    {
+        if (userData?.isAdmin)
+        {
+            return <AdminHeader/>
```

Comment:
Headers should use one shared container for content. So it has same width, height, logo text, colors and so on